packages/vscode-extension/src/officeChat/commands/create/officeCreateCommandHandler.ts

```diff
@@ -70,27 +70,34 @@ export default async function officeCreateCommandHandler(
         ),
       ];
       officeChatTelemetryData.chatMessages.push(...describeProjectChatMessages);
-
       await verbatimCopilotInteraction(
         "copilot-gpt-3.5-turbo",
         describeProjectChatMessages,
         response,
         token
       );
+
       if (matchedResult.type === "sample") {
-        const folder = await showOfficeSampleFileTree(matchedResult, response);
+        const sampleInfos = await showOfficeSampleFileTree(matchedResult, response);
+        const folder = (sampleInfos as any)?.["path"];
+        const hostType = (sampleInfos as any)?.["host"].toLowerCase();
```

Comment:
Use the defined object structure to fetch these variables instead of cast to any

Reply:
will fix, thanks